### PR TITLE
Update examples cluster deployment procedure

### PR DESCRIPTION
**Pull request by jzding**
Signed-off-by: Jack Ding <jacding@redhat.com>

### CODE REVIEW

File: docs/development.md [hunk 1]
```diff
@@ -44,21 +44,30 @@ mv kustomize /usr/local/bin/
...
+oc get pods -n cloud-native-events
 ```
 
+### Deploy examples
+```shell
+make deploy-example
+```
 
-### Delete deployment
+### Undelete examples
```
Comment (aneeshkp) [inline docs/development.md:69]: should be undeploy ?

File: docs/development.md [hunk 2]
```diff
@@ -44,21 +44,30 @@ mv kustomize /usr/local/bin/
...
-export  CONSUMER_IMG = quay.io/aneeshkp/cloud-native-event-consumer
+export version=latest 
+export SIDECAR_IMG=quay.io/aneeshkp/cloud-event-proxy
+export  PRODUCER_IMG=quay.io/aneeshkp/cloud-native-event-producer
+export  CONSUMER_IMG=quay.io/aneeshkp/cloud-native-event-consumer
 ```
 
+### Setup AMQ Interconnect
+
+Install the `Red Hat Integration - AMQ Interconnect` operator in the `cloud-native-events` namespace from the OpenShift Web Console.
```
Comment (aneeshkp) [inline docs/development.md:55]: We should able to deploy amqp to its own namespace. and use the full-service name to access,
amqp-interconnect.NAMESPCE.svc.cluster.local
Need to test this process.

Comment (aneeshkp) [reply docs/development.md:55]: But this is okay for example

**Review by aneeshkp — APPROVED** [review]

File: docs/development.md (hunk 2 above)
Comment (jzding) [reply docs/development.md:55]: will test this

**Pull request merged**